content/post/deeplearning_in_go_part_1.md

```diff
@@ -0,0 +1,393 @@
++++
+title = "Deep Learning from Scratch in Go - Part 1: Equations Are Graphs"
+subtitle = ""
+date = "2017-04-19T08:43:45+10:00"
+latex = true
+draft = true
+
++++
+
+(Author: Chewxy, @chewxy on [Twitter](https://twitter.com/chewxy) and Gophers Slack)
+
+Welcome to the first part of many about writing deep learning algorithms in Go. The goal of this series is to go from having no knowledge at all to implementing a latest paper. 
+
+Deep learning is not new. In fact the idea of deep learning was spawned in the early 1980s. What's changed since then is our computers - they have gotten much much more powerful.  In this blog post we'll start with something familiar, and edge towards building a conceptual model of deep learning. We won't define deep learning for the first few posts, so don't worry so much about the term.
```

Comment:
Some people might be generally unfamiliar with the term deep learning.  Maybe just make "Deep Learning" have a link to some general introduction to the subject?